docs/content/latest/benchmark/tpcc.md

```diff
@@ -0,0 +1,91 @@
+---
+title: TPCC
+linkTitle: TPCC
+description: TPCC
+image: /images/section_icons/architecture/concepts.png
+headcontent: Benchmark YugabyteDB using TPCC.
+menu:
+  latest:
+    identifier: benchmark-tpcc
+    parent: benchmark
+    weight: 740
+aliases:
+  - /benchmark/tpcc/
+showAsideToc: True
+isTocNested: True
+---
+
+Following are the steps to be taken to run TPCC against Yugabyte.
```

Comment:
```suggestion
This page shows you how to run TPCC against YugabyteDB.
```
(Since you are using second person pronouns "you" elsewhere, lets keep the start also consistent).

Reply:
Done.